myapp/spec/controllers/tasks_controller_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
+require 'pp'
```

Comment:
Seemed that we didn't need to require 'pp' as user.
ref: https://qiita.com/scivola/items/b4851f3ebe43a0dba9c5

By the way, where do you use this inside this file?

Reply:
i added this for debugging but i guess it's not needed after debugging, will remove this